site/en/tutorials/keras/basic_text_classification.ipynb

```diff
@@ -335,7 +335,7 @@
         "\n",
         "In this tutorial, we will use the second approach.\n",
         "\n",
-        "Since the movie reviews must be the same length, we will use the [pad_sequences](https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing/sequence/pad_sequences) function to standardize the lengths:"
+        "Since the movie reviews must be the same length, we will use the [pad_sequences](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/preprocessing/sequence/pad_sequences) function to standardize the lengths:"
```

Comment:
Thanks. The TF 1.x preprocessing docs are no longer on the site.

This TF 1.x doc link can point here: https://keras.io/preprocessing/sequence/#pad_sequences

And the TF 2 doc https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/keras/basic_text_classification.ipynb
can point to: https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/preprocessing/sequence/pad_sequences